sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorTransmitter.cs

```diff
@@ -39,30 +39,102 @@ public AzureMonitorTransmitter(AzureMonitorExporterOptions options)
             }
             ConnectionStringParser.GetValues(options.ConnectionString, out _, out string ingestionEndpoint);
             options.Retry.MaxRetries = 0;
-            options.AddPolicy(new IngestionResponsePolicy(this), HttpPipelinePosition.PerCall);
 
             applicationInsightsRestClient = new ApplicationInsightsRestClient(new ClientDiagnostics(options), HttpPipelineBuilder.Build(options), host: ingestionEndpoint);
         }
 
         public async ValueTask<int> TrackAsync(IEnumerable<TelemetryItem> telemetryItems, bool async, CancellationToken cancellationToken)
         {
+            // TODO
+            // Change return type of this function to ExportResult
+            int result = 0;
             if (cancellationToken.IsCancellationRequested)
             {
-                return 0;
+                return result;
             }
 
-            int itemsAccepted = 0;
+            try
+            {
+                using var httpMessage = async ?
+                    await this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false) :
+                    this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).Result;
 
-            if (async)
+                if (httpMessage != null)
+                {
+                    if (storage == null && httpMessage.HasResponse && httpMessage.Response.Status == ResponseStatusCodes.Success)
```

Comment:
Is there a need to check `httpMessage.HasResponse`, we could use `httpMessage.Response?.Status`

Reply:
Yes - it throws an exception otherwise
https://github.com/Azure/azure-sdk-for-net/blob/b1e737580231280ed11f520ccdb2ea86abf69fce/sdk/core/Azure.Core/src/HttpMessage.cs#L38